src/test/resources/mockdata/invoiceLines/invoice_lines.json

```diff
@@ -64,9 +64,9 @@
       ],
       "adjustments": [
         {
-          "adjustmentLabel": "Adjustment label",
-          "adjustmentType": "Percentage",
-          "adjustmentValue": 10.1
+          "description": "Adjustment label",
+          "type": "Percentage",
+          "value": 10.1
```

Comment:
I think these also need the `prorate` field as it's required

Reply:
Done